<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5,6">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>开始时间：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEdit">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="displayFormat">
                 <string>yyyy/M</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>结束时间：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="dateEdit_2">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="displayFormat">
                 <string>yyyy/M</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>班组：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>月份：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_2">
                <property name="font">
                 <font>
                  <family>3ds</family>
                  <pointsize>8</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="font">
               <font>
                <family>3ds</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>连接数据库</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="font">
               <font>
                <family>3ds</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>显示图像</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="title">
            <string>班组-IU均值统计图</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="font">
            <font>
             <family>3ds</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="title">
            <string>班组-月份-IU均值统计图</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="font">
          <font>
           <family>3ds</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>班组-月份-工作辊-IU均值统计图</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
